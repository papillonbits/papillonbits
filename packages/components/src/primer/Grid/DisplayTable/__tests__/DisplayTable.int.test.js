import { regular } from './DisplayTable.int.story'

describe('<DisplayTable />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
