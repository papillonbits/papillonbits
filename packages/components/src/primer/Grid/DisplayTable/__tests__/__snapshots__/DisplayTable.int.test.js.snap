// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayTable /> Render must match regular() 1`] = `
Array [
  <div
    className="d-table col-12 bg-blue-light"
  >
    <div
      className="col-2 d-table-cell border p-2 h6"
    />
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      date-created
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      date-modified
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      kind
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      name
    </div>
    <div
      className="col-2 d-table-cell border p-2 h6"
    >
      size
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Wed May 13 2015
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Fri Feb 03 2017
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      PostScript file
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      y8P.ps
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      37.72 exabyte (EB)
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Fri May 01 2020
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Mon Dec 31 2018
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Perl script file
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Pj9m.cgi
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      339 gigabyte (GB)
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Mon Oct 31 2016
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Tue Feb 21 2017
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      PSD image
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      mRGfhp.psd
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      325.6 kilobyte (KB)
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Wed Mar 11 2020
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Sun Aug 26 2018
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Active Server Page file
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      FBrXhyh736LIV.aspx
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      716 terabyte (TB)
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Fri Mar 21 2014
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Wed Mar 04 2015
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Folder
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      beylpO7v
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      --
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Sun Oct 20 2019
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Fri Feb 08 2019
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Database file
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      aD9iiQhkj.db
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      913.5 bit (b)
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Fri Mar 11 2016
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Thu May 02 2019
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Apple Mail e-mail file
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      9kcUYOj65.emlx
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      921.8 zettabyte (ZB)
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Sun Oct 25 2015
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Thu Apr 11 2019
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Folder
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      7
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      --
    </div>
  </div>,
  <div
    className="d-table col-12"
  >
    <div
      className="col-2 d-table-cell border p-2"
    />
    <div
      className="col-2 d-table-cell border p-2"
    >
      Tue Jan 20 2015
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Tue Oct 15 2013
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      Microsoft Excel file with macros
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      1NplEr0hHjFnTmW.xlsm
    </div>
    <div
      className="col-2 d-table-cell border p-2"
    >
      474.53 petabyte (PB)
    </div>
  </div>,
]
`;
